perllib/FixMyStreet/Cobrand/Northumberland.pm

```diff
@@ -292,6 +294,65 @@ sub record_update_extra_fields {
     };
 }
 
+=head2 inform_about_shortlisted
+
+We send email and text to inspector when report (re)allocated to them.
+
+=cut
+
+sub inform_about_shortlisted {
+    my ( $self, $user, $report ) = @_;
+
+    # XXX In future we may want to do some extra filtering here, to only send
+    # to users with specific roles or
+    # for reports with specific categories
+
+    my $email = $user->email;
+    my $phone = $user->phone;
+
+    if ($email) {
```

Comment:
It's probably not needed, but do we want to add an && email_verified here just in case?